code/datums/uplink_item.dm

```diff
@@ -715,6 +715,13 @@ var/list/uplink_items = list()
 	cost = 5
 	surplus = 90
 
+/datum/uplink_item/stealthy_tools/mulligan
```

Comment:
If you wanna rename it might as well rename all the spots where it says mulligan to randomizer